.github/scripts/automergeCrowdinPr.js

```diff
@@ -81,21 +97,20 @@ module.exports = async ({ github, context }) => {
       event: 'APPROVE',
       body: `Approved from [${context.workflow} #${context.runNumber}](${context.serverUrl}/${context.repo.owner}/${context.repo.repo}/actions/runs/${context.runId}).`,
     })
-  } else {
-    console.log(`Already reviewed`)
   }
 
-  const hasAutomergeLabel = pr.labels.some((label) => label.name === AUTOMERGE_LABEL)
-  if (!hasAutomergeLabel) {
-    console.log(`Adding ${AUTOMERGE_LABEL} label`)
-    await github.rest.issues.addLabels({
-      owner,
-      repo,
-      issue_number: pr.number,
-      labels: [AUTOMERGE_LABEL],
-    })
-  } else {
-    console.log(`Already labelled with ${AUTOMERGE_LABEL}`)
-  }
+  console.log(`Bringing PR #${pr.number} up to date with main branch`)
+  await github.rest.pulls.updateBranch({
```

Comment:
i tried to achieve feature parity with mergify here but i think there's still a reasonable chance for the PR's to become out of date by the time the CI finishes and therefore not merged (mergify would merge main into the branches again automatically and now we'd be doing it manually, but let's see how it goes)

Reply:
Great 👍

Reply:
oh this is great